### PR TITLE
(feat #70) Navbar configuration:

### PR DESCRIPTION
**Pull request by AlDu2407**
- Removed fixed links in Urchin.
- Added configurable `links` property in Urchin configuration.
- Updated application settings to include new links property.
- Updated `templ` to add additional parameter for links array.
- Updated `app_settings_test`.

(fix) Makefile:

- Fixed version numbers in `install-tools`.

### CODE REVIEW

File: urchin_config.toml [hunk 1]
```diff
@@ -19,3 +19,11 @@ webserver_port = 8080
 
 # Directory to use for storing uploaded images.
 image_dir = "./images"
+
+links = [
```
Comment (matheusgomes28) [inline urchin_config.toml:23]: I like it, but I think we should have a `[navbar]` section here, something like

```toml
[navbar]
page= "/"
links = [
...
]
```

This way we could change navbar links per page. However, don't suggest adding the page in this PR, just put the `[nvabar]` section (by doing an extra inner struct in the config struct), then we can implement extensions in the future 😄

**Review by matheusgomes28 — COMMENTED** [review]
Looks good Al, just need to increase the coverage now for tests.

I'm gonna start enforcing the coverage as we've had PRs from other people in the last couple of weeks that are forgetting to cover the newly added lines in tests, just so we at least have some sort of regression testing.

File: urchin_config.toml (hunk 1 above)
Comment (AlDu2407) [reply urchin_config.toml:24]: Added `[navbar]` as suggested.

**Review by matheusgomes28 — APPROVED** [review]
LGTM!

**Pull request merged**